docs/source/_static/vendor/sphinx-footer.css

```diff
@@ -0,0 +1,749 @@
+.footer-h5 {
```

Comment:
I do not think you need such a big css footer. All this code is more maintenance. Please can you reduce it to the essential? thank you